examples/03_connectivity/plot_seed_based_correlation.py

```diff
@@ -155,9 +155,9 @@
 
 # Finally, we can tranform the correlation array back to a Nifti image
 # object, that we can save.
-seed_based_correlation_img = brain_masker.inverse_transform(
-    seed_based_correlations.T)
-seed_based_correlation_img.to_filename('sbc_z.nii.gz')
+seed_based_correlations_fisher_z_img = brain_masker.inverse_transform(
+    seed_based_correlations_fisher_z.T)
+seed_based_correlations_fisher_z_img.to_filename('sbc_z.nii.gz')
```

Comment:
I would call the file `seed_based_correlation_z.nii.gz` or even better,  `pcc_based_correlation_z.nii.gz`